scripts/setupvars/setupvars.sh

```diff
@@ -58,7 +58,7 @@ if [ -e $INSTALLDIR/deployment_tools/inference_engine ]; then
     fi
 
     export KMB_INSTALL_DIR=$INSTALLDIR/deployment_tools/inference_engine/external/hddl_unite
-    export LD_LIBRARY_PATH=$KMB_INSTALL_DIR/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
+    export LD_LIBRARY_PATH=$KMB_INSTALL_DIR/lib:$KMB_INSTALL_DIR/thirdparty/XLink/lib${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
```

Comment:
So it will implicitly override XLink library installed to the system, isn't it? Won't there be any problems?

Reply:
This change is about x86 platform. We don't expect any XLink libraries installed in that case. If it is installed for some reason, we need to use the one from the package.  

Actually, KMB_INSTALL_DIR is not needed to be set for AARCH64 platform (LD_LIBRARY_PATH related changes not needed as well). I can check if the OS_NAME is ubuntu then set KMB_INSTALL_DIR. Is this check OK?

Reply:
> I can check if the OS_NAME is ubuntu then set KMB_INSTALL_DIR. Is this check OK?

I don't think it is necessary. Maybe it would be better to check if this or some other directory exists? Is there a difference between AARCH64 and x86 platforms in package contents?

Reply:
Yes. There is. Your suggestion makes sense to me. I will update the script accordingly.

Reply:
Updated.